packages/url/src/index.js

```diff
@@ -34,6 +34,51 @@ export function addQueryArgs( url, args ) {
 	return baseUrl + '?' + stringify( { ...query, ...args } );
 }
 
+/**
+ * Returns a single query argument of the url
+ *
+ * @param  {string} url URL
+ * @param  {string} arg Query arg name
+ *
+ * @return {Array|string} Query arg value.
+ */
+export function getQueryArg( url, arg ) {
+	const queryStringIndex = url.indexOf( '?' );
```

Comment:
Given repetition in the file, seems we could do for an internal utility function(s) which return the parsed query and/or return the query fragment from a string.

Reply:
Yeah it's not really nice at the moment. I think we should address this in a following PR.